CHANGELOG.md

```diff
@@ -3,7 +3,6 @@
 This is a feature release.
 
 * [PCP-786](https://tickets.puppetlabs.com/browse/PCP-786) Send error response to status if found
```

Comment:
I'm guessing that PCP-771 will be added to the CHANGELOG once we figure out the solaris failures?

Reply:
Yeah, we'll put it under the appropriate version.